OpenRVS/classes/OpenMultiPlayerWidget.uc

```diff
@@ -165,18 +167,35 @@ function GetGSServers()
 		NewItem.Created();
 		NewItem.iMainSvrListIdx = i;
 		NewItem.bFavorite = true;
-		NewItem.bSameVersion = true;
+		NewItem.bSameVersion = !ServerList[i].bWrongVersion;//true;//1.5 change
 		NewItem.szIPAddr = ServerList[i].IP;
-		NewItem.iPing = 1000;//was 1000 in early versions, at some point post 0.6 was changed to 9000?
+		if ( ( ServerList[i].iPing != 0 ) && ( ServerList[i].iPing != -1 ) && ( ServerList[i].iPing != 1000 ) )//1.5 change
+			NewItem.iPing = ServerList[i].iPing;
+		else
+		{
+			ServerList[i].iPing = 1000;//for sorting - null value is 0 which sorted as better than low ping servers
+			NewItem.iPing = 1000;
+		}
 		NewItem.szName = ServerList[i].ServerName;
-		NewItem.szMap = "";
-		NewItem.iMaxPlayers = 0;
-		NewItem.iNumPlayers = 0;
+		NewItem.szMap = ServerList[i].szMap;//"";//1.5 change
+		if ( ServerList[i].iMaxPlayers == 0 )//1.5 change

```

Comment:
Ints default to 0, but I want servers that didn't respond to sort worse than servers with 0/16 players.  So this logic sets servers not responding to -1 players so it always sorts worse than servers that respond.

Reply:
next to the `iMaxPlayers ==0` line can you comment something like `server did not respond`? took me a minute to parse the reason for checking it at first